tests/search/components/TestSearchForm.js

```diff
@@ -7,24 +7,30 @@ const render = ReactTestUtils.renderIntoDocument;
 const findByTag = ReactTestUtils.findRenderedDOMComponentWithTag;
 
 describe('<SearchForm />', () => {
+  var onSearch;
```

Comment:
We could - I'm on the fence about that. I'm a bit too used to using `var` and I've only been using let when I need it. Maybe we should just use let and const and be done with it? Happy to hear thoughts on that -  I can go with the consensus and adapt.